src/Scheduler.ts

```diff
@@ -17,7 +21,7 @@ import { Subscription } from './Subscription';
  *
  * @class Scheduler
  */
-export class Scheduler {
+export class Scheduler implements IScheduler {
 
   public static now: () => number = Date.now ? Date.now : () => +new Date();
```

Comment:
I believe schedulers were at one point static classes in Rx.NET, but could be misremembering.